src/imagery/i.asf.download/testsuite/token.txt

```diff
@@ -0,0 +1 @@
+eyJ0eXAiOiJKV1QiLCJvcmlnaW4iOiJFYXJ0aGRhdGEgTG9naW4iLCJzaWciOiJlZGxqd3RwdWJrZXlfb3BzIiwiYWxnIjoiUlMyNTYifQ.eyJ0eXBlIjoiVXNlciIsInVpZCI6ImNvcGVybmljdXNudmUiLCJleHAiOjE2OTExNDYxNDAsImlhdCI6MTY4NTk2MjE0MCwiaXNzIjoiRWFydGhkYXRhIExvZ2luIn0.vBmSTqH0nKcJ3VQ9khXijYgv1f57GCVfhpnvr4Y4tI3irFZa5njoLHhDMb4TXCi4-gQCN-gz8nVJNNtlZ9X2_9v2L5UlJ7dLkeL-OeH_bAC__K8SXr5YAewsYF9Z_pFBGnMFVyDvaPbfD6kox1pxuXgPj8ItM16l4kTtr8xBYC-PNJolbMbilNeUie7tY0Sl5cU5P6p54zgGar-qcBX13TLX9otcHo5C0PIU5NHj0h_tzRlIVhXJmIaUeVLLXJhm-gAb-aEqQV9DCiKNj6m1QnYeAUK4-XRuVVICyi8m6LwpthIcXF2J0IZw-C1Y8elUUmCR7k6_-J8W5UgvTIjEyA
```

Comment:
This token is basically compromised...
The file is not needed and can be removed if we add a github secret ASF_TOKEN and use it in CI as environement variable (if you go for my suggestion for token (file) handling...